### DISCUSSION TITLE
v0.0.5

### DISCUSSION THREAD

**Discussion by EdoFro** [EdoFro/Freeplane_Map-Drive-Inator#10] [Announcements]
## v0.0.5

* corrected **createFilesFromNotes** functionality

* Added '**Import MDI styles**' functionality (\*)
_(\*) without library: only works for users that installed other add-ons that have the `edofro.pseudofreeplaneapi.UserStyles` library._

* Added **_exception handling_** for cases when mindmaps doesn't have **MDI styles** yet

<hr /><em>This discussion was created from the release <a href='https://github.com/EdoFro/Freeplane_Map-Drive-Inator/releases/tag/v0.0.5'>v0.0.5</a>.</em>

**Comment by EdoFro** [comment]
> * Added '**Import MDI styles**' functionality (*)
>   _(*) without library: only works for users that installed other add-ons that have the `edofro.pseudofreeplaneapi.UserStyles` library._

currently, the add-ons, that have the `edofro.pseudofreeplaneapi.UserStyles` library, are:
- Tutorial-o-Matic
- groovyNode
- MarkdownHelper

If you installed any of these then this functionality should work in Freeplane for you.

**Reply by EdoFro** [reply]
You can install the library by just installing the add-on [Freeplane_Pseudo-Freeplane-API](https://github.com/EdoFro/Freeplane_Pseudo-Freeplane-API)

the add-on can be downloadesd from [its release page](https://github.com/EdoFro/Freeplane_Pseudo-Freeplane-API/releases/latest)

[pseudoFreeplaneAPI-v0.0.1.addon.mm](https://github.com/EdoFro/Freeplane_Pseudo-Freeplane-API/releases/download/v0.0.1/pseudoFreeplaneAPI-v0.0.1.addon.mm)

**Reply by bepolymathe** [reply]
Thanks. It’s a good and more easy option for those have not MarkdownHelper installed...
